tests/scripts/github-action-helper.sh

```diff
@@ -735,7 +735,7 @@ function install_minikube_with_none_driver() {
   rm "$CNI_PLUGIN_TAR"
 
   export MINIKUBE_HOME=$HOME CHANGE_MINIKUBE_NONE_USER=true KUBECONFIG=$HOME/.kube/config
-  sudo -E minikube start --kubernetes-version="$1" --driver=none --memory 6g --cpus=2 --addons ingress --cni=calico
+  minikube start --kubernetes-version="$1" --driver=none --memory 6g --cpus=2 --addons ingress --cni=calico
```

Comment:
why is `sudo` removed?

Reply:
@sp98 it was giving error
```
X Exiting due to HOST_HOME_CHOWN: Failed to change permissions for /home/runner: Error changing ownership for: /home/runner: chown /home/runner/.
``` that Parth mentioned.